src/libs/actions/Wallet.js

```diff
@@ -262,13 +262,6 @@ function openEnablePaymentsPage() {
     API.read('OpenEnablePaymentsPage');
 }
 
-/**
- * @param {Object} keyValuePair
- */
-function updateAdditionalDetailsDraft(keyValuePair) {
-    Onyx.merge(ONYXKEYS.WALLET_ADDITIONAL_DETAILS_DRAFT, keyValuePair);
```

Comment:
Looks like `ONYXKEYS.WALLET.ADDITIONAL_DETAILS_DRAFT` is no longer used, so we should remove it from ONYXKEYS.js

Reply:
I was thinking similar, but aren't we actually still trying to access this data here? https://github.com/Expensify/App/blob/6c74c2cf8595ca1ba30306dfd5451f77ad7369f7/src/pages/EnablePayments/OnfidoStep.js#L39-L41

Reply:
So, the `Form` element will automatically create a "draft" ONYX object for the `WALLET_ADDITIONAL_DETAILS`. However, the `WALLET_ADDITIONAL_DETAILS` object is used in more places than just AdditionalDetailsStep.js. It is also used places like IdologyQuestions.js, which does not use the `Form` element, so the draft is not automatically created. I think we have to keep ONYXKEYS.WALLET.ADDITIONAL_DETAILS_DRAFT for now...but I could be misunderstanding something.

Reply:
@joelbettner Sorry for the really long delay in replying to this.

> So, the Form element will automatically create a "draft" ONYX object for the WALLET_ADDITIONAL_DETAILS

Agree

> However, the WALLET_ADDITIONAL_DETAILS object is used in more places than just AdditionalDetailsStep.js ... [and in those other places] ... the draft is not automatically created

I also agree with this, but the `WALLET_ADDITIONAL_DETAILS_DRAFT` key that was being manually managed by the `additionalDetailsStep` was only being _consumed_ by the `additionalDetailsStep`. So since `WALLET_ADDITIONAL_DETAILS_DRAFT` isn't being used anywhere anymore, we should be able to get rid of it.